yash-env/CHANGELOG.md

```diff
@@ -29,7 +32,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 
 - The implementation of `From` for converting `errno::Errno` to and from
-  `nix::errno::Errno`.
+  `nix::errno::Errno`
+- The `getopts_state` field from the `Env` struct
```

Comment:
Should also note that the GetoptsState and Origin definitions have been removed.

Reply:
```suggestion
- The `getopts_state` field from the `Env` struct
- The `builtin::getopts` module and its contents
  (the `GetoptsState` struct and the `Origin` enum)
```